tex/base.tex

```diff
@@ -78,17 +78,17 @@ \subsection*{RV32I Base Integer Instructions}
 or       & OR                      & R   & 0110011    & 0x6    & 0x00   & rd = rs1 | rs2               & \\
 and      & AND                     & R   & 0110011    & 0x7    & 0x00   & rd = rs1 \& rs2              & \\
 sll      & Shift Left Logical      & R   & 0110011    & 0x1    & 0x00   & rd = rs1 \verb|<<| rs2       & \\
-srl      & Shift Right Logical     & R   & 0110011    & 0x2    & 0x00   & rd = rs1 \verb|>>| rs2       & \\
-sra      & Shift Right Arith*      & R   & 0110011    & 0x3    & 0x20   & rd = rs1 \verb|>>| rs2       & msb-extends \\
-slt      & Set Less Than           & R   & 0110011    & 0x2    &        & rd = (rs1 < rs2)?1:0         & \\
-sltu     & Set Less Than (U)       & R   & 0110011    & 0x3    &        & rd = (rs1 < rs2)?1:0         & zero-extends \\ \hline
-addi     & ADD Immediate           & I   & 0010011    & 0x0    & 0x00   & rd = rs1 + imm               & \\
-xori     & XOR Immediate           & I   & 0010011    & 0x4    & 0x00   & rd = rs1 \^{} imm            & \\
-ori      & OR Immediate            & I   & 0010011    & 0x6    & 0x00   & rd = rs1 | imm               & \\
-andi     & AND Immediate           & I   & 0010011    & 0x7    & 0x00   & rd = rs1 \& imm              & \\
-slli     & Shift Left Logical Imm  & I   & 0010011    & 0x1    & 0x00   & rd = rs1 \verb|<<| imm       & \\
-srli     & Shift Right Logical Imm & I   & 0010011    & 0x1    & 0x00   & rd = rs1 \verb|>>| imm       & \\
-srai     & Shift Right Arith Imm   & I   & 0010011    & 0x3    & 0x20   & rd = rs1 \verb|>>| imm       & msb-extends \\
+srl      & Shift Right Logical     & R   & 0110011    & 0x5    & 0x00   & rd = rs1 \verb|>>| rs2       & \\
+sra      & Shift Right Arith*      & R   & 0110011    & 0x5    & 0x20   & rd = rs1 \verb|>>| rs2       & msb-extends \\
+slt      & Set Less Than           & R   & 0110011    & 0x2    & 0x00   & rd = (rs1 < rs2)?1:0         & \\
+sltu     & Set Less Than (U)       & R   & 0110011    & 0x3    & 0x00   & rd = (rs1 < rs2)?1:0         & zero-extends \\ \hline
+addi     & ADD Immediate           & I   & 0010011    & 0x0    &        & rd = rs1 + imm               & \\
+xori     & XOR Immediate           & I   & 0010011    & 0x4    &        & rd = rs1 \^{} imm            & \\
+ori      & OR Immediate            & I   & 0010011    & 0x6    &        & rd = rs1 | imm               & \\
+andi     & AND Immediate           & I   & 0010011    & 0x7    &        & rd = rs1 \& imm              & \\
+slli     & Shift Left Logical Imm  & I   & 0010011    & 0x5    & imm[10]=0 & rd = rs1 \verb|<<| imm[0:4]  & \\
```

Comment:
I think the value of `func3` is wrong here (it should be 0x1)

Reply:
Looks like it, yeah. V20191213 of the spec lists `slli` as:
```
0000000 shamt rs1 001 rd 0010011
```

Also, any reason for the `imm[10]=0`, if it's all zeroes and in the above it's listed as `0x00`?